plugins/linux-spi-lpc/fu-plugin-linux-spi-lpc.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
+
+#define FU_PLUGIN_LINUX_SPI_LPC_SYSFS_DIR	"/sys/kernel/security/spi"
```

Comment:
I think this should go into fu-common stuff

Reply:
Ya, lets get the kernel bits upstream first :)

Reply:
Oh I guess the point being to add the path after it's upstream, yeah makes sense.